heroku-api-integration-tests/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>heroku-api-parent</artifactId>
         <groupId>com.heroku.api</groupId>
-        <version>0.17-SNAPSHOT</version>
+        <version>3.0.0-SNAPSHOT</version>
```

Comment:
That's a pretty big jump. Is `3.0` used because you want to match the API versioning?